app/src/main/java/net/aiscope/gdd_app/ui/policy/PrivacyPolicyActivity.kt

```diff
@@ -0,0 +1,118 @@
+package net.aiscope.gdd_app.ui.policy
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.webkit.JavascriptInterface
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.annotation.WorkerThread
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.isVisible
+import dagger.android.AndroidInjection
+import net.aiscope.gdd_app.R
+import net.aiscope.gdd_app.databinding.ActivityPrivacyPolicyBinding
+import net.aiscope.gdd_app.ui.main.MainActivity
+import javax.inject.Inject
+
+
+class PrivacyPolicyActivity : AppCompatActivity(), PrivacyPolicyView {
+
+    @Inject
+    lateinit var presenter: PrivacyPolicyPresenter
+
+    private lateinit var binding: ActivityPrivacyPolicyBinding
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override fun onCreate(savedInstanceState: Bundle?) {
+        AndroidInjection.inject(this)
+        super.onCreate(savedInstanceState)
+        binding = ActivityPrivacyPolicyBinding.inflate(layoutInflater)
```

Comment:
Nice! Is there a way to prevent the synthetics creation?

Reply:
You mean the kotlinx ones? Yes, by removing the `kotlin-android-extensions` Gradle plugin, but we would have to migrate to View Binding everywhere first :))

Reply:
Nice! Thanks! :-)